providers/implementations/rands/drbg.c

```diff
@@ -183,17 +183,23 @@ static size_t prov_drbg_get_entropy(PROV_DRBG *drbg, unsigned char **pout,
         if (buffer != NULL) {
             size_t bytes = 0;
 
+            if (drbg->parent_generate == NULL)
+                goto err;
```

Comment:
Oh, I overlooked this fix. It's not mentioned in the commit. Maybe even add it as a separate commit?

Reply:
It's not a fix.  The diff decided that "moving" this is easier than fixing the comment.
I moved this before the comment so try to get it (the comment) closer to the line warned about.

Reply:
Sorry, you are right. I was confused by the diff output.